packages/odyssey-react/src/components/FieldLabel/FieldLabel.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { Story } from "@storybook/react";
+import React from "react";
+import FieldLabel from ".";
+import type { Props } from ".";
+
+export default {
+  title: `Components/FieldLabel`,
+  component: FieldLabel,
+  argTypes: {
+    children: { control: 'text' },
+    id: { control: 'text' },
+  },
+  parameters: {
+    controls: {
+      sort: 'requiredFirst'
+    }
+  }
+};
+
+const Template: Story<Props> = (props) => (
+  <FieldLabel {...props} />
+);
+
+export const Label = Template.bind({});
+Label.args = {
+  variant: "label",
+  children: "Destination"
+};
```

Comment:
Not the worst thing - but selecting the `optional` variant knob from any label outside of the "Optional" story doesn't end up rendering the text, as `<span data-optional>` is missing.

Reply:
heh I noticed this too 😅. Taking a step back, having the visibility of the label (or really just the optional label) entirely controlled by the presence of a class (or data attribute) on a sibling element does not make sense if `FieldLabel` is going to be consumed as its own component. This violates the modular design principles we are aspiring too and provides a confusing and hidden API for developers. There are plans to discuss form elements in depth next sprint so for now I'm not too worried as we'll be revisiting this all soon. cc @edburyenegren-okta @arnoldsandoval-okta @joejezowski-okta